Formula/k2cli.rb

```diff
@@ -0,0 +1,17 @@
+class K2cli < Formula
+  desc "CLI for K2"
+  homepage "https://github.com/samsung-cnct/k2cli"
+  url "https://github.com/samsung-cnct/k2cli/releases/download/1.0.5/k2cli_1.0.5_darwin_amd64.tar.gz"
```

Comment:
It's our policy to always allow users the option to build from source.  We generate binaries from our source builds and offer those as well (as "bottles").

As for the test, since the formula `depends_on` docker, it will be available during the test so you can rely on it for writing a more comprehensive test.